packages/react-aria-components/src/Menu.tsx

```diff
@@ -399,6 +399,7 @@ function MenuItemTriggerInner<T>({item, popover, parentMenuRef, delay}: MenuItem
           triggerRef: ref,
           placement: 'end top',
           UNSTABLE_portalContainer: submenuContext.popoverContainerRef.current || undefined,
+          containerPadding: 0,
```

Comment:
Open question about this one, do we want to just hard code this to 0 or let the user configure it? This is what it looks like when using the default (12px):
<img width="577" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/09745515-b92e-4a0b-aef4-8df4c0deaed7">

Note the offset applied to the 3rd menu, this is because we are offsetting by that additional amount

This is with the above applied:
<img width="517" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/dd6d84f0-b349-4c68-bd5c-fe8da25d00cc">

Reply:
Depending on the way the menus are styled (such as if there is additional margins and stuff pushing the menu up/down), this could still look a bit off...

Reply:
in the first one you mean it's actually at 24px?

Reply:
hmm, actually on a closer look, they are both offset by 12px, its something with how the menus are styled in the storybook. I'll revert, but perhaps `containerPadding` is something to still consider providing to the user.